fn/promise/index.js

```diff
@@ -0,0 +1,2 @@
+require('../../modules/es7.promise.finally');
+module.exports = require('../../modules/_core').Promise;
```

Comment:
> We already have `fn/promise.js`
OK, `require`d es7.promise.finally in `fn/promise.js` in leu of `fn/promise/finally.js`. Or do you want them both, so they can just add finally?